index.html

```diff
@@ -1,108 +1,159 @@
+      <a href="?year=2011">2011</a>
+    </nav>
+    <!-- NavBar END -->
+
+      <div class="content row">
+
+        {{# if participants}}
+          <div class="row">
+            <h2 class="content"> <b class="largeText">{{ participants }}</b> participants </h2>
+          </div>
+        {{/if}}
+
+        {{#if participants}}
+          <div class="col-sm-9">
+            <div class="col-sm-5">
+              <h2> Distributions </h2>
+              <canvas id="distributions" height="400"></canvas>
+            </div>
+
+            {{# if facultyData}}
+              <div class="col-sm-5">
+                <h2> Faculty </h2>
+                <canvas id="faculty" height="400"></canvas>
+              </div>
+            {{/if}}
+
+            <div class="col-sm-12">
+              <h2> Helpers </h2>
+              <canvas id="helpers" height="400"></canvas>
+            </div>
+
+            <div class="col-sm-12">
+              <h2> Participants <h2>
+              <input id="participantsFilter" type="text" placeholder="Search.."></input>
+              <div id="participants"></div>
+            </div>
+          </div>
+
+          <div id="partners" class="col-sm-3">
+                <h2>Partners</h2>
+                {{#each partners}}
+                <a href={{this.url}}> <img class={{this.type}} src={{this.src}} /> </a>
+                {{/each}}
+                <iframe src="http://www.facebook.com/plugins/likebox.php?href=https%3A%2F%2Fwww.facebook.com%2Frosedu.org&amp;width=200&amp;height=290&amp;colorscheme=light&amp;show_faces=true&amp;header=true&amp;stream=false&amp;show_border=false" scrolling="no" frameborder="0" style="border:none; overflow:hidden; width:200px; height:290px;" allowTransparency="true"></iframe>
+              </div>
+
+        {{else}}
+            <div class="description col-sm-9">
+              <strong>ROSEdu</strong> pregătește cea de a 10-a ediție a
+              <a href="https://www.facebook.com/events/569614456477589/"><strong>Linux Install Fest</strong></a>.
+              <div>
+                Ai posibilitatea, alături de noi, să îți instalezi orice distribuție Linux pe laptopul sau computerul tău.
+                Vom fi alături de tine pentru a rezolva problemele legate de drivere, partiționare sau migrarea datelor între sisteme de operare diferite.
+              </div>
+              <div>
+                <strong>Linux Install Fest</strong> își propune să aducă împreună timp de câteva ore oameni pasionați din diverse comunități Linux.
+              </div>
+              <div class="note">
+                *Dacă ai un sistem desktop îți poți aduce doar unitatea. Noi vom pune la dispoziție monitoare, tastaturi și mouse-uri.
+              </div>
+              <div>
+                <strong>Când</strong>
+              </div>
+              <div class="time">
+                Sâmbătă, <strong> 10 Octombrie </strong> - Între <strong> 10:00 și 16:00 </strong>.
```

Comment:
Si astea ar trebui sa fie variabile incarcate in template, probabil intr-un alt issue